pkg/pgclient/config.go

```diff
@@ -46,13 +48,17 @@ func ParseFlags(cfg *Config) *Config {
 	flag.Uint64Var(&cfg.MetricsCacheSize, "metrics-cache-size", pgmodel.DefaultMetricCacheSize, "Maximum number of metric names to cache.")
 	flag.IntVar(&cfg.WriteConnectionsPerProc, "db-writer-connection-concurrency", 4, "Maximum number of database connections for writing per go process.")
 	flag.IntVar(&cfg.MaxConnections, "db-connections-max", -1, "Maximum number of connections to the database that should be opened at once. It defaults to 80% of the maximum connections that the database can handle.")
+	flag.StringVar(&cfg.ServiceUrl, "service-url", DefaultServiceUrlFromDefaultFlags, "TimescaleDB/Vanilla Postgres service URL.")
```

Comment:
2 things:
1. Postgres calls this a URI not a URL, so lets change it to URI. Also, should we call this `db-uri` not service... This would fall in-line with the other db- parameters.
2. As mentioned before, I think an empty-string default is better. But we should move the current default into the description string as an example.